docs/semgrep-ci/running-semgrep-ci-with-semgrep-app.md

```diff
@@ -60,12 +61,11 @@ Support for certain features of Semgrep Cloud Platform may depend on your CI pro
 ## Setting up the CI job and Semgrep Cloud Platform connection
 
 ![Steps to run Semgrep in CI without Semgrep Cloud Platform](/img/semgrep-ci-overview-app.png "Steps to integrate Semgrep in CI with Semgrep Cloud Platform")
-
 *Figure 1.* Steps to run Semgrep in CI with Semgrep Cloud Platform.
 
-Refer to the succeeding sections for guidance specific to your CI provider.
+The next sections provide guidance for specific CI provider.
 
-### CI providers listed within Semgrep Cloud Platform (such as GitHub Actions, GitLab CI/CD, Jenkins)
+### CI providers listed in Semgrep Cloud Platform
```

Comment:
@s-santillan I am shortening the header, as brevity might be more than overwhelming information (there is a list and a screenshot below anyway - I also fixed some broken links (some were still mentioning In-App). We were also using CI providers listed in Semgrep Cloud Platform often in some other link descriptions...